### DISCUSSION TITLE
K1 Extruder Mounts

### DISCUSSION THREAD

**Discussion by brentdeacon23** [3DCoded/3MS#34] [General]
Anyone with a K1 have a good setup for mounting the extruders? Maybe onto the back of the machine? Trying to think of the best layout for it all

**Comment by 3DCoded** [comment]
Hi @brentdeacon23 ,

For the K1, you have three main options:

1. [3MS Enclosure by **chadk**](https://www.printables.com/model/1067703-wip-3ms-mmu-using-btt-mmb). This doesn't have mounting holes, but it does keep the wires organized if you use a BTT MMB CAN board.
2. [Minimalistic M3 Mount](https://3dcoded.github.io/3MS-docs/setup/assembly/#mounting). This lets you mount your extruders to an aluminum extrusion.
3. Design your own mount.
